internal/schema/schema.go

```diff
@@ -340,6 +343,52 @@ func (s *Schema) Parse(schemaString string, useStringDescriptions bool) error {
 		}
 	}
 
+	for _, iface := range s.interfaces {
+		iface.Interfaces = make([]*Interface, len(iface.interfaceNames))
+
+		for i, intfName := range iface.interfaceNames {
+			t, ok := s.Types[intfName]
+			if !ok {
+				return errors.Errorf("interface %q not found", intfName)
+			}
+			intf, ok := t.(*Interface)
+			if !ok {
+				return errors.Errorf("type %q is not an interface", intfName)
+			}
+			for _, f := range intf.Fields.Names() {
+				if iface.Fields.Get(f) == nil {
+					return errors.Errorf("interface %q expects field %q but %q does not provide it", intfName, f, iface.Name)
+				}
+			}
+
+			iface.Interfaces[i] = intf
+		}
+	}
+
+	for _, iface := range s.interfaces {
+		if iface == nil {
+			continue
+		}
+		expectedNames := interfaceDependancies(*iface)
+		actualNames := iface.interfaceNames
+
+		expMap := make(map[string]int)
+		actMap := make(map[string]int)
```

Comment:
Could you, please, remove the `Map` suffix from the names. From the syntax below it is obvious that it is a map.